### ISSUE TITLE
After linux compile: WindNinja_cli can't find libgdal.so.20

### ISSUE THREAD

**Issue by tcarternws** [firelab/windninja#403]
Running in a non-admin user environment WindNinja_cli reports:  

WindNinja_cli: error while loading shared libraries: libgdal.so.20: cannot open shared object file: No such file or directory

The gdal libraries are in location /usr/local/lib and this is in the user path. The install location is in /usr/local/bin for WindNinja_cli

I can run the build version of WindNinja_cli if I cd to /usr/local/src/wind/build/src/cli . 

Where my my build install steps have gone wrong for WN to run for system users?

Thanks.

**Comment by nwagenbrenner** [comment]
It sounds like whoever installed WindNinja to the system location with root privileges needs to run sudo ldconfig to update the shared libraries links.

**Comment by tcarternws** [comment]
I installed and ldconfig was run right after I rebuilt WindNinja but the issue continued..

Apparent solution:
ldconfig -p showed none of the WindNinja dependency libraries in the ld.so.cache

I added a locallibs.conf to /etc/ld.so.conf.d that contained a/usr/local/libs  entry

ldconfig then picks up the many additional libraries and adds them to the cache

We are on Redhat 7 enterprise server. Apparently Unbutu handles the cache of new libraries better after install?  I'm not entirely understanding if the library is in the user path why WindNinja could not find it but having the libraries in the ld cache works.

Anyway now the cli runs for a non-admin user in their home directory. Thanks for the response.
